frontend/client/src/screens/Login.js

```diff
@@ -58,9 +58,9 @@ const SignInFormBase = observer(
       // Based on https://firebase.google.com/docs/auth/web/email-link-auth
       if (this.props.store.isSignInWithEmailLink(window.location.href)) {
         Logging.log('signInWithEmailLink detected')
+        var email = new URL(location.href).searchParams.get('email')
         // Get the email if available. This should be available if the user completes
         // the flow on the same device where they started it.
-        var email = window.localStorage.getItem('emailForSignIn')
         if (!email) {
```

Comment:
This is dead code. Magic links will always have an `email` query string parameter.